app/androidutils.cpp

```diff
@@ -35,3 +35,24 @@ bool AndroidUtils::isAndroid() const
     return false;
 #endif
 }
+
+void AndroidUtils::requirePermissions()
+{
+    checkAndAcquirePermissions("android.permission.WRITE_EXTERNAL_STORAGE");
+    checkAndAcquirePermissions("android.permission.CAMERA");
```

Comment:
I think we should only ask for camera permission the first time the user is going to take a picture... otherwise it is kind of scare to give an application access to camera immediately when it starts :-)

Reply:
Camera permission has been removed. It's automatically requested when PhotoPanel is init, thus when featureForm with externalResource field is init. Would be better to send a signal only if a new picture is about to be taken.